include/Core/UI/Setting/EmailLogSetting.php

```diff
@@ -0,0 +1,113 @@
+<?php
+namespace EmailLog\Core\UI\Setting;
+use EmailLog\Core\UI\Setting\SettingField;
+
+class EmailLogSetting extends Setting {
+
+	/**
+	 * Implement `initialize()` method.
+	 */
+	protected function initialize() {
+		$this->section->id          = 'email-log';
+		$this->section->title       = __( 'Plugin Settings', 'email-log' );
+		$this->section->option_name = 'el_email_log';
+
+		$this->load();
+	}
+
+	/**
+	 * Implement `get_fields()` method.
+	 *
+	 * @return array
+	 */
+	public function get_fields() {
+		$fields = array();
+
+		$email_log_fields = array(
+			'allowed_user_roles'  => __( 'Allowed User Roles', 'email-log' ),
+		);
+
+		foreach ( $email_log_fields as $field_id => $label ) {
+			$field           = new SettingField();
+			$field->id       = $field_id;
+			$field->title    = $label;
+			$field->args     = array( 'id' => $field_id );
+			$field->callback = array( $this, 'render_email_log_capability_field' );
```

Comment:
Let's use seperate render function for each field.

Reply:
Fixed.